### PR TITLE
Docs(web): Improve `Accordion` README and demo preview

### PR DESCRIPTION
**Pull request by adamkudrna**

### CODE REVIEW

**Review by literat — CHANGES_REQUESTED** [review]
Nitpicking, anything else looks great.

File: packages/web/src/scss/components/Accordion/README.md [hunk 1]
```diff
@@ -1,88 +1,206 @@
 # Accordion
 
-Accordion is a wrapper for multiple [**Collapse**][collapse] items, and can be open one or more at time.
+Accordion is a composition of multiple [**Collapse**][collapse] items. Depending
```
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web/src/scss/components/Accordion/README.md:3]: TextLint will tell us what's right 🙂.

[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web/src/scss/components/Accordion/README.md:3]: Actually not Text Lint, but MD lint… It's a question of personal preference, linters are fine with it. But I fixed it so we don't spend any more time talking about it 🙂.

File: packages/web/src/scss/components/Accordion/README.md [hunk 2]
```diff
@@ -1,88 +1,206 @@
...
+
+```html
+<section class="Accordion" data-toggle="accordion">
+  <!-- One ore more items -->
+</section>
+```
+
+### Accordion Item
+
+Put one or more accordion items into the accordion wrapper:
```
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web/src/scss/components/Accordion/README.md:29]: OK, thanks for pointing this out. There are more places, I fixed them all.

**Review by literat — APPROVED** [review]

File: packages/web/src/scss/components/Accordion/README.md [hunk 3]
```diff
@@ -1,88 +1,206 @@
...
-        <div class="Accordion__content">content</div>
+<section id="accordion" class="Accordion" data-toggle="accordion">
+  <!-- Accordion items here… -->
+</section>
+```
+
+Then link it using the `data-parent` attribute on individual Collapse items:
+
+```html
+<article class="Accordion__item"></article>
```
Comment (adamkudrna) [inline packages/web/src/scss/components/Accordion/README.md:183]: Fix the closing tag

File: packages/web/src/scss/components/Accordion/README.md [hunk 4]
```diff
@@ -1,88 +1,206 @@
...
+Depending on your needs, one or more Accordion items can be open at a time.
 
-## Usage
+## Basic Usage
 
-### Default (Always open)
+Building blocks:
+
+- Accordion
+  - Accordion item (one or more)
```
Comment (adamkudrna) [inline packages/web/src/scss/components/Accordion/README.md:11]:
```suggestion
  - Accordion item (one or, typically, more)
```

**Pull request merged**